Makefile

```
@@ -153,6 +154,14 @@ update-node: ## Update a single node (NODE=name) from CONFIG=<path>
 	@[ -n "$(NODE)" ]   || { echo 'NODE=<name> required (CP_NAME or one of WORKER_NAMES)' >&2; exit 2; }
 	@CONFIG="$(CONFIG)" NODE="$(NODE)" sudo -E bash scripts/update-cluster.sh --node="$(NODE)"
 
+export-argocd: ## Export an ArgoCD-registerable kubeconfig (OUTPUT=, SERVER=, CONTEXT=, FORCE=1)
```

checkmake (0.3.2)

[warning] Target body for "export-argocd" exceeds allowed length of 5 lines (6).

(maxbodylength)